CONTRIBUTING.md

```diff
@@ -0,0 +1,48 @@
+# Contributing code
+
+The fundamental code standard we use
+[StandardJS](http://standardjs.com/rules.html), the rules being
+described in the aforementioned link. We highly recommend you stick to
+this standard to ease the maintenance for us.
+
+1. Fork the repository.
+2. Create a new branch, which very briefly summarizes the change you're going to make.
+
+   For example:
+   
+   - Fixing a typo? ==> `typo`
+   - Documentation addition/fix/etc such as adding the documentation
+     for a method ==> `doc/method-<method name>` or a class ==> `doc/class-<class name>`.
```

Comment:
method/class is a java concept and we won't be using it much - maybe use different examples here?